io.ts

```diff
@@ -54,12 +50,39 @@ export async function sendCommand(
   command: string,
   ...args: (number | string)[]
 ): Promise<RedisRawReply> {
-  const msg = createRequest(command, ...args);
+  const msg = createRequest(command, args);
   await writer.write(encoder.encode(msg));
   await writer.flush();
   return readReply(reader);
 }
 
+export async function sendCommands(
+  writer: BufWriter,
+  reader: BufReader,
+  commands: {
+    command: string;
+    args: (number | string)[];
+  }[],
+): Promise<RawReplyOrError[]> {
+  const msg = commands.map((c) => createRequest(c.command, c.args)).join("");
+  await writer.write(encoder.encode(msg));
+  await writer.flush();
+  const ret: RawReplyOrError[] = [];
+  for (let i = 0; i < commands.length; i++) {
+    try {
+      const rep = await readReply(reader);
+      ret.push(rep);
+    } catch (e) {
+      if (e instanceof ErrorReplyError) {
+        ret.push(["error", e]);
```

Comment:
```suggestion
        const reply = ["error", e];
        ret.push(reply);
        commands[i].d.reject(reply);
```

Reply:
Can this variable be `rep` just to be consistent with the above?